airflow/api_connexion/openapi/v1.yaml

```diff
@@ -3813,6 +3813,24 @@ components:
           type: string
           description: The plugin source
           nullable: true
+        ti_deps:
+          type: array
+          items:
+              type: string
+              nullable: true
```

Comment:
When can these be null instead of an empty list?

Reply:
Is there another way to represent the empty list? I thought marking it nullable meant the list was empty.

Reply:
I think _without_ nullable would just work, an empty list is allowed by default. To disallow an empty array you’d need to set something else (`minLength: 1` or something, don’t remember)